src/js/modules/infragistics.ui.widget.js

```diff
@@ -98,36 +99,93 @@
 				return $.ig.regional.defaults[ key ];
 			}
 		},
+		_getLocaleWidgetName: function () {
+			this.localeWidgetName = this.localeWidgetName ||
+							(this.widgetFullName || this.widgetName)
+								.replace(/^((ui-ig(\.)*)|(ig(\.)*))/ig, "");
+			return this.localeWidgetName;
+		},
+		_getLocaleObject: function (locale) {
+			return $.ig.locale &&
```

Comment:
This method does not allow extracting a value from the locale option provided to the specific instance of the control.